### DISCUSSION TITLE
win10修改了hosts文件内容后蓝灯一直闪退

### DISCUSSION THREAD

**Discussion by xiaomilao** [getlantern/lantern-forum-cn#284] [Q&A]
win10系统早上因为解析需要在本地hosts中加了一条记录，然后蓝灯就无法启动了。一打开就闪退，重新清除软件下载蓝灯还是同样的问题，不知道该如何解决

**Comment by casperwzy** [comment]
不是因为你修改，这样的问题，我什么都没动，自动就闪退了

**Comment by MaxMKMax** [comment]
@casperwzy @xiaomilao 嗯嗯. 今天有突发性的问题导致闪退. 估计有新的bug. 而且很多人遇到这个问题. 我明白的. 我马上帮你们反馈.
